### PR TITLE
Migrate to Update 11

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose

$Subject

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "oauth2"
-version = "2.12.1"
+version = "2.13.0"
```
Comment (NipunaRanasinghe) [inline ballerina/Ballerina.toml:4]: I guess we also need to update the distribution version in L10?

**Comment by NipunaRanasinghe** [comment]
@ThisaruGuruge shall we also update the gradle version in the gradle wrapper properties file? in other modules we updated it to 8.11.x versions

**Comment by ThisaruGuruge** [comment]
> @ThisaruGuruge shall we also update the gradle version in the gradle wrapper properties file? in other modules we updated it to 8.11.x versions

Sorry, missed this. I will update.

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (ThisaruGuruge) [reply ballerina/Ballerina.toml:4]: Fixed with [31f4d9a](https://github.com/ballerina-platform/module-ballerina-oauth2/pull/1330/commits/31f4d9aeda3e6e10d3278371a970503acc1db2b6)

**Review by RDPerera — APPROVED** [review]

**Pull request merged**